rapid/plugin/x/mysqlxtest_src/mysqlx_protocol.cc

```diff
@@ -159,6 +159,8 @@ bool mysqlx::parse_mysql_connstring(const std::string &connstring,
   return true;
 }
 
+static void throw_server_error(const Mysqlx::Error &error) MY_ATTRIBUTE((noreturn));
+
 static void throw_server_error(const Mysqlx::Error &error)
```

Comment:
Can't you just merge declaration and definition?

Reply:
Looks like I can, I just had to add MY_ATTRIBUTE earlier. I changed it.